docs/docs/airgap.md

```diff
@@ -15,11 +15,11 @@ In this scenario, you will need a QA/pre-production XCP-ng pool with Internet ac
 
 ### Deployment
 
-[Follow the standard procedure](https://xen-orchestra.com/docs/installation.html) to deploy the XOA on your pool with Internet access.
+[Follow the standard procedure](installation) to deploy the XOA on your pool with Internet access.
```

Comment:
There might something to fix in config of docurausus of elsewhere, short links are not as expected : 
Should be `https://docs.xen-orchestra.com/installation`
Is `https://xen-orchestra.com/docs/installation.html`

![image](https://github.com/user-attachments/assets/c3609e23-2807-43be-b619-a45642b2af62)